net-print/brother-mfc7820n-bin/brother-mfc7820n-bin-2.0.1-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils rpm linux-info
+
+DESCRIPTION="Brother printer driver for MFC-7820N"
+
+HOMEPAGE="http://support.brother.com"
+
+MY_PR=${PR/r/}
+
+SRC_URI="http://download.brother.com/welcome/dlf006265/brmfc7820nlpr-${PV}-${MY_PR}.i386.rpm
+        http://download.brother.com/welcome/dlf006267/cupswrapperMFC7820N-${PV}-${MY_PR}.i386.rpm"
+
+
```

Comment:
Also a comment from repoman. Remove the empty line. One empty line is enough. :)